### PR TITLE
Update code to use wp_star_rating() function

### PR DESCRIPTION
**Pull request by jazbek**
Update code to use [wp_star_rating()](https://codex.wordpress.org/Function_Reference/wp_star_rating) function

Updates #35818

### CODE REVIEW

File: src/admin-views/tribe-options-help.php [hunk 1]
```diff
@@ -184,9 +184,11 @@
 
 	<?php if ( isset( $rating ) && isset( $num_rating ) ) { ?>
 		<h3><?php esc_html_e( 'Average Rating', 'tribe-events-calendar' ); ?></h3>
-		<div class="star-holder">
-			<div class="star star-rating" style="width: <?php echo esc_attr( $rating ); ?>px"></div>
-		</div>
+		<?php wp_star_rating(array(
```
Comment (borkweb) [inline src/admin-views/tribe-options-help.php]: Slight spacing tweak: missing a space between `wp_star_rating(` and `array(`:

```
wp_star_rating( array(
```

**Comment by borkweb** [comment]
After that spacing tweak, this looks :+1:

**Comment by jazbek** [comment]
Whole file needed a spacing tweak. :)

**Comment by borkweb** [comment]
Whee! Merge it :D

**Pull request merged**